packages/charts/examples/components/demo/chart-types/area/area-chart-bi-directional-stacked/area-chart-bi-directional-stacked-example.component.html

```diff
@@ -25,16 +25,16 @@
                            (mouseenter)="chartAssist.emphasizeSeries(legendSeries.id)"
                            [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state"
                            [isSelected]="!chartAssist.isSeriesHidden(legendSeries.id)"
-                           (isSelectedChange)="chartAssist.toggleSeries(legendSeries.id, $event)">
+                           (isSelectedChange)="onSelectedChange(legendSeries, $event, chartAssist)">
 
             <!-- Specify the 'tick' formatter to display positive values for both series in the legend. -->
             <!-- Specify the 'y1' data accessor to let the chart assist know how to access the data for the legend. -->
             <nui-rich-legend-tile [value]="chartAssist.getHighlightedValue(legendSeries, 'y', 'tick', 'y1')"
-                                  [backgroundColor]="chartAssist.palette.standardColors.get(legendSeries.id)"
-                                  [color]="chartAssist.palette.textColors.get(legendSeries.id)">
+                                  [backgroundColor]="chartAssistTop.palette.standardColors.get(legendSeries.id)"
+                                  [color]="chartAssistTop.palette.textColors.get(legendSeries.id)">
                 <nui-chart-marker rightEdge
-                                  [marker]="chartAssist.markers.get(legendSeries.id)"
-                                  [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
+                                  [marker]="chartAssistTop.markers.get(legendSeries.id)"
+                                  [color]="chartAssistTop.palette.standardColors.get(legendSeries.id)">
```

Comment:
Can the two chart assists share the markers and palette instances so that we can just use `chartAssist` here instead of `chartAssistTop`?